zio-http/src/main/scala/zio/http/Response.scala

```diff
@@ -75,7 +72,7 @@ final case class Response private (
   def withServerTime: Response = self.copy(attribute = self.attribute.withServerTime)
 
   private[zio] def close: Task[Unit] = self.attribute.channel match {
-    case Some(channel) => ChannelFuture.unit(channel.close())
+    case Some(channel) => NettyFutureExecutor.unit(channel.close())
```

Comment:
I think it would be nice to rename this `unit` too, it was very confusing for me first

Reply:
Done,  Renamed to `executed`